OOInteraction/src/commands/CAddCalleesToView.cpp

```diff
@@ -71,11 +70,15 @@ Interaction::CommandResult* CAddCalleesToView::executeWithArguments(Visualizatio
 				view->insertNode(ancestor->node(), 0, 0);
 				pos = view->positionOfNode(ancestor->node());
 			}
-			//TODO@cyril Insert the nodes in a good order
 			view->insertColumn(pos.x() + 1);
-			auto row = callees_.size() - 1;
+			auto row = 0;
+			//Make the first callee appear at the same height as the method
+			if (height > 10) //Higher than margin?
+			{
+				view->addEmptyNode(pos.x() + 1, 0, height);
```

Comment:
We should really just have one type of node (e.g. ViewItemNode). The method here could be called just addSpacer (wihout the Node suffix). It is then an implementation detail that a spacer is a node.